test/parallel/test-net-localerror.js

```diff
@@ -26,13 +26,13 @@ const net = require('net');
 
 connect({
   host: 'localhost',
-  port: common.PORT,
```

Comment:
This should not be changed. Perhaps a comment should be added indicating that because the connection is never actually made, there's no chance of a port collision so using `common.PORT` is OK.